packages/clay-icon/src/ClayIcon.soy

```diff
@@ -6,11 +6,23 @@
 {template .render}
 	{@param spritemap: string}
 	{@param symbol: string}
+	{@param? ariaLabel: string}
+	{@param? decorative: bool}
 	{@param? elementClasses: string}
 	{@param? focusable: bool}
 	{@param? id: string}
 	{@param? title: string}
 
+	{let $containerAttributes kind="attributes"}
+		{if $decorative}
+			aria-hidden="true"
+		{/if}
+
+		aria-label="{$ariaLabel ?: $symbol}"
+		class="lexicon-icon-container"
```

Comment:
`elementClasses` should be applied to the first element of the component, so here.

Reply:
Wouldn't this be a breaking change since we are moving classes that were once on the SVG to the span?

Reply:
Pattern we're following places the elementClasses in the first element always. IMHO markup change is already a breaking change so let's try to be consistent at least with the pattern :)